ironfish/src/syncer.ts

```diff
@@ -355,12 +358,17 @@ export class Syncer {
         )} (${sequence}) from ${peer.displayName}`,
       )
 
+      const start = Date.now()
```

Comment:
Maybe use BenchUtils.start for this? It's vaguely more accurate, not that it makes a huge difference, and we're using it elsewhere in syncer

Reply:
👍🏻